### PR TITLE
[Improvement] Dragging ability of inspector popover

### LINKED ISSUE #3433: Able to drag and reposition the leftSidebar popover
## Description
Ability to reposition the LeftSidebar Inspector popover anywhere on the screen via dragging the popover.

### PR DESCRIPTION
**Pull request by arpitnath**
Resolves #3433

### CODE REVIEW

**Comment by shubh22** [comment]
Task linked: [CU-2hcxyvh Able to Drag and Reposition the LeftSidebar popover](https://app.clickup.com/t/2hcxyvh)

File: frontend/src/_styles/theme.scss [hunk 1]
```diff
@@ -325,7 +325,7 @@ button {
   .editor-sidebar {
     height: 100%;
     position: fixed;
-    z-index: 1;
+    // z-index: 1;
```
Comment (kavinvenkatachalam) [inline frontend/src/_styles/theme.scss:328]: If not needed, Could you please remove this?

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@arpitnath Few findings

- I moved the inspector and resized it. Then when I try to expand the items, the content overflows out of the parent window. 
image

- Opened `CodeHinter` in a popup and tried opening an `Inspector` nothing happens

**Comment by Nandinisaha13** [comment]
<html>
<body>
<google-sheets-html-origin>Test Cases

Test Cases | Steps | Result
-- | -- | --
Should be able to move inspector. | Create an application | Application gets created.
  | Click on inspector. | Inspector opens up.
  | Drag anywhere on the screen. | Inspector is dragged.
Should be able to close inspector. | Create an application | Application gets created.
  | Click on inspector. | Inspector opens up.
  | Click on cross button. | The inspector is closed.
Should be able to view inspector in light mode. | Create an application | Application gets created.
  | Click on inspector. | Inspector opens up.
  | Observe. | Inspector is clearly visible.
Should be able to view inspector in dark mode. | Create an application | Application gets created.
  | Click on dark mode icon. | Application is visible in dark mode
  | Click on inspector. | Inspector opens up.
  | Observe. | Inspector is clearly visible.
Should be able to expand the inspector popover. | Create an application | Application gets created.
  | Click on inspector. | Inspector opens up.
  | Expand the popover. | Inspector popover is expanded correctly.
Should be able to drag inspector popover. | Create an application | Application gets created.
  | Click on inspector. | Inspector opens up.
  | Drag inspector popover. | Inspector pop over is dragged anywhere on the screen.


</body>
</html>

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@arpitnath Few findings and suggestions

- Opened the `Inspector` and without closing it I opened the `CodeHinter` in a popup. Then I closed the `CodeHinter` and tried opening the `Inspector`, but I don't see the `Inspector`
- You can see in the screenshot that if I try to expand multiple components then it goes beyond the viewport height and when I try to scroll, the page header is not fixed. (Suggestion - Can we add a fixed height to the `Inspector` and enable the overflow to it?) cc: @sherfin94 
image

**Review by kavinvenkatachalam — APPROVED** [review]
LGTM

**Review by sherfin94 — CHANGES_REQUESTED** [review]
Hey @arpitnath , I'm not able to keep the inspector open and make changes at the same time. This feature is heavily used to see data changes in real time when components are interacted with.

**Pull request closed without merging**